components/alert.html

```diff
@@ -0,0 +1,229 @@
+                        A simple primary alert—check it out!
+                      </div>
+                      <div class="alert alert-secondary" >
+                        A simple secondary alert—check it out!
+                      </div>
+                      <div class="alert alert-success" >
+                        A simple success alert—check it out!
+                      </div>
+                      <div class="alert alert-danger" >
+                        A simple danger alert—check it out!
+                      </div>
+                      <div class="alert alert-warning" >
+                        A simple warning alert—check it out!
+                      </div>
+                      <div class="alert alert-info" >
+                        A simple info alert—check it out!
+                      </div>
+                      <div class="alert alert-light" >
+                        A simple light alert—check it out!
+                      </div>
+                      <div class="alert alert-dark" >
+                        A simple dark alert—check it out!
+                      </div>
+                </div>
+
+                <div class="component-block-html">
+                    <div class="block-html">HTML CODE</div>
+                    <pre>
+                       <code class="language-html">
+                            &lt;div class="alert alert-primary"&gt;
+                                A simple primary alert—check it out!
+                            &lt;/div &gt;
+
+                            &lt;div class="alert alert-secondary"&gt;
+                                A simple secondary alert—check it out!
+                            &lt;/div&gt;
+
+                            &lt;div class="alert alert-success"&gt;
+                                A simple success alert—check it out!
+                            &lt;/div&gt;
+
+                            &lt;div class="alert alert-danger"&gt;
+                                A simple danger alert—check it out!
+                            &lt;/div&gt;
+
+                            &lt;div class="alert alert-warning"&gt;
+                                A simple warning alert—check it out!
+                            &lt;/div&gt;
+
+                            &lt;div class="alert alert-info"&gt;
+                                A simple info alert—check it out!
+                            &lt;/div&gt;
+
+                            &lt;div class="alert alert-light"&gt;
+                                A simple light alert—check it out!
+                            &lt;/div&gt;
+
+                            &lt;div class="alert alert-dark"&gt;
+                                A simple dark alert—check it out!
+                            &lt;/div&gt;
+
+                       </code>
+                   </pre>
+
+                </div>
+                
+                <!-- <div class="component-block-css">
```

Comment:
I think you can remove the commented part from the main code. If its of no relevance .